test/extended/idling/idling.go

```diff
@@ -216,7 +216,7 @@ var _ = g.Describe("[sig-network-edge][Feature:Idling]", func() {
 	})
 
 	g.Describe("Idling", func() {
-		g.Context("with a single service and DeploymentConfig", func() {
+		g.Context("with a single service and DeploymentConfig [apigroup:route.openshift.io]", func() {
```

Comment:
also apps.openshift.io

Reply:
apps.openshift.io is already present in g.It:
```
g.It("should idle the service and DeploymentConfig properly [apigroup:apps.openshift.io]", func() {
```
do we need to add it here as well?

Reply:
We don't need to given the fact.